03 - Clustering Misorientations.ipynb

```diff
@@ -165,120 +246,296 @@
     "cluster_means = cluster_means.set_symmetry(D6, D6)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Inspect cluster means in quaternion representation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cluster_means"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Inspect cluster means in axis-angle representation"
+   ]
+  },
   {
    "cell_type": "code",
-   "execution_count": 9,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "array([57.32484832, 68.81388421, 31.58939741, 60.10115054])"
-      ]
-     },
-     "execution_count": 9,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
+   "outputs": [],
    "source": [
     "np.degrees(cluster_means.angle.data)"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Generate map\n",
-    "mapping = np.ones(misori_base.shape + (3,))\n",
-    "\n",
-    "mapping[np.where(boundary_mask)[0][~small_mask], np.where(boundary_mask)[1][~small_mask]] = c"
+    "cluster_means.axis"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Define reference misorientations associated with twinning orientation relationships"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from orix.vector import Vector3d\n",
+    "from orix.quaternion.rotation import Rotation\n",
+    "from orix.vector.neo_euler import AxAngle"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# From Krakow et al\n",
+    "sigma7a = Rotation.from_neo_euler(AxAngle.from_axes_angles((1, 0, 0), np.radians(64.40)))\n",
+    "sigma11a = Rotation.from_neo_euler(AxAngle.from_axes_angles((1, 0, 0), np.radians(34.96)))\n",
+    "sigma11b = Rotation.from_neo_euler(AxAngle.from_axes_angles((2, 1, 0), np.radians(85.03)))\n",
+    "sigma13a = Rotation.from_neo_euler(AxAngle.from_axes_angles((1, 0, 0), np.radians(76.89)))\n",
+    "sigma13b = Rotation.from_neo_euler(AxAngle.from_axes_angles((2, 1, 0), np.radians(57.22)))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# From Laine et al\n",
+    "sigma7a = Rotation.from_neo_euler(AxAngle.from_axes_angles((1, 0, 0), np.radians(64.40)))\n",
+    "sigma11a = Rotation.from_neo_euler(AxAngle.from_axes_angles((1, 0, 0), np.radians(34.96)))\n",
+    "sigma11b = Rotation.from_neo_euler(AxAngle.from_axes_angles((1, 1, 0), np.radians(85.03)))\n",
```

Comment:
I'm a bit confused, why is this `sigma11b` off a different axis `(1,1,0)` to the `sigma11b` from Krakow `(2,1,0)`?

Reply:
I think this is part of the minefield of different conventions for hexagonal systems and also that symmetry relationships with 3-index notation are not so obvious as with 4-index for hexagonal systems...